docs/application/tizen-studio/native-tools/ui-builder/custom-ui-component.md

```diff
@@ -23,7 +23,7 @@ A custom component requires an EDJ resource file (`[ComponentName].edj`) and an
 
     Each custom UI component has an EDJ resource file. The custom UI component makes this EDJ file available to the **Design** editor by placing it on the **Palette** and enables visual editing in the **Properties** view.
 
-    The EDJ file is created by compiling EDC code with the [EDC Editor](edc_editor_n.htm). For more information about EDC, see [Layouting with EDC](../../../org.tizen.guides/html/native/ui/efl/learn_edc_intro_n.htm).
+    The EDJ file is created by compiling EDC code with the [EDC Editor](edc-editor.md). For more information about EDC, see [Layouting with EDC]( ../../native/guides/ui/efl/learn-edc-intro.md).
```

Comment:
It looks `../edc-editor.md` and `../../../native/guides/ui/efl/learn-edc-intro.md` are correct.

Reply:
done